producer-cpp/docker-rtsp/Dockerfile

```diff
@@ -29,9 +23,24 @@ RUN apt-get upgrade && \
 WORKDIR /opt/
 RUN git clone https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp.git
 WORKDIR /opt/amazon-kinesis-video-streams-producer-sdk-cpp/build/
-RUN cmake .. -DBUILD_GSTREAMER_PLUGIN=ON &&\
+RUN cmake .. -DBUILD_GSTREAMER_PLUGIN=ON && \
     make
 
+
+FROM ubuntu:18.04
+
+RUN apt update && \
```

Comment:
Quick question. 
We would need the gstreamer install steps in the BUILDER right and not here?
Wouldnt the build `cmake .. -DBUILD_GSTREAMER_PLUGIN=ON && \` fail because gstreamer is not installed? Have we tried building it after the new changes?